cms/static/js/certificates/models/certificate.js

```diff
@@ -20,6 +20,8 @@ function (_, str, Backbone, BackboneRelational, BackboneAssociations, gettext, C
         defaults: {
             // Metadata fields currently displayed in web forms
             course_title: '',
+            course_number: '',
+            display_coursenumber: '',
 
```

Comment:
@saleem-latif  can you explain why we need to introduce these attributes in certificate model ? It seems me that we need to show the course number and its override (if applicable) for read-only purpose.